### PR TITLE
Album cover: Album artist fallback option

### PR DESCRIPTION
**Pull request by LeadRDRK**
Added an option for falling back to album artist if album cover couldn't be found using the track's artist. Works on tracks which album artist and track artist differ.

### CODE REVIEW

**Comment by kuba160** [comment]
I had a discussion about it on deadbeef discord and the way it should be done is to pick "artist", "album artist", "composer" or "performer" (first available). If you could implement it inside `for` loop it would be great. There is no need to add another option in plugin properties. Cheers.

**Comment by LeadRDRK** [comment]
I implemented it in the latest commit. Everything should be good now.

File: discord_presence.c [hunk 1]
```diff
@@ -215,16 +215,24 @@ static void updateDiscordPresence(int playback_status, float song_len) {
...
+        
+        if (lastfm_album[0]) {
+            char lastfm_artist[MAX_LEN];
+            for (int i = 0; i < ARTIST_KEY_COUNT; ++i) {
+                *lastfm_artist = 0;
+                deadbeef->pl_get_meta(nowplaying, artist_keys[i], lastfm_artist, MAX_LEN);
+                if (lastfm_artist[0]) {
+                    int ret = fetch_from_lastfm(lastfm_artist, lastfm_album, lastfm_link, MAX_LEN);
+                    if (ret > 0)
+                        discordPresence.largeImageKey = lastfm_link;
```
Comment (kuba160) [inline discord_presence.c:234]: Shouldn't there be `break` when artist/album pair is correct and we found `lastfm_link`?

Comment (kuba160) [reply discord_presence.c:234]: I also noticed that actions aren't run on pull requests, gonna push a commit fixing it now.

Comment (LeadRDRK) [reply discord_presence.c:234]: Oops, missed that. Fixed it now.

**Pull request merged**